packages/cli/internal/pkg/cli/config/io.go

```diff
@@ -25,9 +29,16 @@ func fromYaml(filePath string) (Config, error) {
 		return Config{}, err
 	}
 	var configData Config
+	setDefault(&configData)
 	if err := yaml.Unmarshal(bytes, &configData); err != nil {
 		return Config{}, err
 	}
-
 	return configData, nil
 }
+func setDefault(configData *Config) Config {
```

Comment:
This approach currently has a bug in it where if the yaml file doesn't exist (or has an error), then Config that is returned won't include the required default text value.

You should probably change the function "toYaml" so that it sets a default Config value (since it now has possible defaults) and then override them with values from the config file.